package-lock.json

```diff
@@ -12923,7 +12923,7 @@
       "dependencies": {
         "make-plural": {
           "version": "4.3.0",
-          "resolved": "https://registry.npmjs.org/make-plural/-/make-plural-4.3.0.tgz",
+          "resolved": "http://registry.npmjs.org/make-plural/-/make-plural-4.3.0.tgz",
```

Comment:
https > http downgrade?

Reply:
that's just what my npm install does, nobody knows why...